microcache_test.go

```diff
@@ -366,6 +365,23 @@ func TestNoWriteHeader(t *testing.T) {
 	cache.Stop()
 }
 
+// Stop
+func TestStop(t *testing.T) {
+	cache := New(Config{})
+	done := make(chan bool)
+	go func() {
+		cache.Stop()
+		done <- true
+	}()
+	select {
+	case <-time.After(100 * time.Millisecond):
+		t.Log("Middleware failed to stop")
+		t.Fail()
```

Comment:
You could collapse these two lines like so:
```golang
t.Fatal("Middleware failed to stop")
```

Reply:
Done. I'm also going to refactor other tests to make this consistent.